app/events/db/db_event_handler.py

```diff
@@ -266,3 +266,8 @@ async def dictionary_brands_event_handler(cls, data: Dict):
     async def dictionary_locations_event_handler(cls, data: Dict):
         from app.events.db.mapper_config import DICTIONARY_LOCATIONS_TABLE_MAPPINGS
         await cls.persist_to_db(table_mappings=DICTIONARY_LOCATIONS_TABLE_MAPPINGS, data=data.get('data'))
+
+    @classmethod
+    async def profile_contact_info_event_handler(cls, data: Dict):
+        from app.events.db.mapper_config import PROFILE_CONTACT_INFO_TABLE_MAPPINGS
+        await cls.persist_to_db(table_mappings=PROFILE_CONTACT_INFO_TABLE_MAPPINGS, data=data)
```

Comment:
wouldnt it be
await cls.persist_to_db(table_mappings=PROFILE_CONTACT_INFO_TABLE_MAPPINGS, data=data.get('data')

Reply:
we use `data.get('data')` when the input data contains another dictionary or payload under a specific key (in this case, 'data') that needs to be extracted and passed to the `persist_to_db` method.
In current case, the POST API is providing the input data that is already in the correct format required by the table mapping and hence we simply pass the `data`.